node/core/approval-voting/src/lib.rs

```diff
@@ -428,7 +431,8 @@ struct ApprovalVoteRequest {
 
 #[derive(Default)]
 struct Wakeups {
-	// Tick -> [(Relay Block, Candidate Hash)]
+	// Tick -> [(Relay Block, Vec of Candidate Hash)]
+	// For Compact modulo VRF wakeups we want to wake-up once for all candidates
```

Comment:
Comment is changed here but behavior is not?

Though I don't think the behavior needs to change

Reply:
I fixed the comment back to initial form, nothing was changed.